packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/FieldLayoutOptions.tsx

```diff
@@ -117,59 +112,63 @@ const FieldLayoutOptions: FC<Props> = ({ items }) => {
 
     return (
         <>
-            <AxisGroup>
-                <AxisTitleWrapper>
-                    <AxisTitle>
+            <Stack spacing="xs" mb="lg">
+                <Group position="apart">
+                    <Text fw={500} sx={{ alignSelf: 'end' }}>
                         {`${
                             validCartesianConfig?.layout.flipAxes ? 'Y' : 'X'
                         }-axis`}
-                    </AxisTitle>
-                    <SimpleButton
-                        text="Flip axes"
+                    </Text>
+                    <Button
+                        variant="subtle"
+                        compact
                         onClick={() => setFlipAxis(!dirtyLayout?.flipAxes)}
-                    />
-                </AxisTitleWrapper>
+                    >
+                        Flip axes
+                    </Button>
+                </Group>
                 {dirtyLayout?.xField === EMPTY_X_AXIS ? (
                     <Button
-                        minimal
-                        intent="primary"
+                        variant="subtle"
+                        compact
+                        sx={{
+                            alignSelf: 'start',
+                        }}
                         onClick={() => setXField(getItemId(items[0]))}
                     >
                         + Add
                     </Button>
                 ) : (
-                    <AxisFieldDropdown>
+                    <Group spacing="xs">
                         <FieldSelect
                             selectedField={xAxisField}
                             fieldOptions={items}
                             onChange={(newValue) => {
-                                setXField(newValue ?? undefined);
+                                setXField(newValue ?? '');
```

Comment:
im still not sure about this. can we have this instead `setXField(newValue ?? undefined)` and zselectedField={xAxisField ?? ''}`

Reply:
Sure. I imagine you know something about our internals that I don't 😄 . React and Mantine are both pretty opinionated about using the empty string as the empty value for inputs, so we probably should do this translation everywhere or investigate whether we are comfortable using empty string. But I'm happy to do this for now.

Reply:
selectedField looks like it can't be undefined, so I will just change this callback.